timur/lib/client/jsx/components/attributes/attribute_viewer.jsx

```diff
@@ -18,73 +18,49 @@ import MarkdownAttribute from './markdown_attribute';
 
 import PlotAttribute from './plot_attribute';
 
-export default class AttributeViewer extends React.Component{
-  render(){
+const AttributeViewer = (props) => {
+  let {attribute_name, record, revision={}, model_name, template, component } = props;
+  let attribute = template.attributes[attribute_name];
 
-    let {attribute} = this.props;
-
-    switch(attribute.attribute_class){
-      case 'LinePlotAttribute':
-      case 'BoxPlotAttribute':
-      case 'BarGraphAttribute':
-      case 'TimelineGroupPlotAttribute':
-      case 'BarPlotAttribute':
-      case 'StackedBarPlotAttribute':
-      case 'SwarmAttribute':
-      case 'HistogramAttribute':
-      case 'PlotAttribute':
-        return <PlotAttribute {...this.props} />;
-
-      case 'DemographicAttribute':
-      case 'DiagnosticAttribute':
-      case 'TreatmentAttribute':
-      case 'AdverseEventAttribute':
-        return <ClinicalAttribute  {...this.props} />;
-
-      case 'BoxPlotAttribute':
-        return <BoxPlotAttribute {...this.props} />;
-      case 'TextAttribute':
-        return <TextAttribute {...this.props} />;
-      case 'MarkdownAttribute':
-        return <MarkdownAttribute {...this.props} />;
-      case 'MetricsAttribute':
-        return <MetricsAttribute {...this.props} />;
+  if (!attribute) {
+    console.log(`No such attribute ${attribute_name} in template for ${model_name}`);
```

Comment:
Nit; make it a console.error so it's easier to notice?  Especially because test runners by default filter logs, but not errors, from output.